test/common/unit/libraries/MessageLib.t.sol

```diff
@@ -280,16 +280,27 @@ contract TestMessageLibIdentities is Test {
         // This message is a submessage and has not static message length defined
     }
 
-    function testUpdateContractMaxPriceAge() public pure {
-        MessageLib.UpdateContractMaxPriceAge memory a =
-            MessageLib.UpdateContractMaxPriceAge({assetId: 1, maxPriceAge: 42});
-        MessageLib.UpdateContractMaxPriceAge memory b = MessageLib.deserializeUpdateContractMaxPriceAge(a.serialize());
+    function testUpdateContractMaxAssetPriceAge() public pure {
+        MessageLib.UpdateContractMaxAssetPriceAge memory a =
+            MessageLib.UpdateContractMaxAssetPriceAge({assetId: 1, maxPriceAge: 42});
+        MessageLib.UpdateContractMaxAssetPriceAge memory b =
+            MessageLib.deserializeUpdateContractMaxAssetPriceAge(a.serialize());
```

Comment:
Jeroen has updated these tests to be fuzzed, maybe it's interesting to fuzz this now to not miss them later (fully optional!)

Reply:
You are talking about #234 which is still WIP, right?